### PR TITLE
ostree: add page

### PR DESCRIPTION
**Pull request by fwilhe2**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** v2022.7

### CODE REVIEW

**Comment by fwilhe2** [comment]
Hi everyone,

I tried to stick to the guidelines, please let me know if I did anything wrong. I know this is a similar case to git where it could make sense to make a page per sub-command, but I wanted to keep it simple to start with. I'm happy to improve upon this, but I think this page is already valuable to anyone diving into ostree (at least I would have wished for this page to exist, that's why I wrote it :) )

File: pages/linux/ostree.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# OSTree
+
+> Version control for binary files similar to git but optimized for operating system root filesystems.
+> OSTree is the foundation for immutable image-based operating systems such as Fedora Silverblue, Fedora IoT or Fedora CoreOS.
+> More information: <https://ostreedev.github.io/ostree/>.
```
Comment (blueskyson) [inline pages/linux/ostree.md:5]:
```suggestion
> More information: <https://ostreedev.github.io/ostree>.
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution. Here are some slight suggestions.

**Review by kbdharun — APPROVED** [review]
Wow, OSTree command finally 🚀 . Welcome to tldr and Thanks for your contribution.

LGTM after @blueskyson's comments is addressed.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**